modules/gdscript/gdscript_editor.cpp

```diff
@@ -1115,10 +1115,12 @@ static void _find_identifiers_in_class(const GDScriptParser::ClassNode *p_class,
 							continue;
 						}
 						option = ScriptLanguage::CodeCompletionOption(member.function->identifier->name, ScriptLanguage::CODE_COMPLETION_KIND_FUNCTION, location);
-						if (member.function->parameters.size() > 0) {
+						if (member.function->parameters.size() > 0 || bool((member.function->info.flags & METHOD_FLAG_VARARG) >> 4)) {
```

Comment:
What's the deal with this right shift? Wouldn't `(member.function->info.flags & METHOD_FLAG_VARARG)` do the same?